File: cfgs/java_cfgs_foo/FIND_THE_FIRST_MISSING_NUMBER.java
Nodes
1;special;;;
2;conditional;foo_1 > foo_2;14;
3;statement;return foo_1 + 1;14;
4;exit;;;
5;conditional;foo_1 != foo_2[foo_3];15;
6;statement;return foo_1;15;
7;statement;int mid = (foo_1 + foo_2) / 2;16;
8;conditional;foo_1[foo_2] == foo_3;17;
9;statement;return fun_1(foo_1, foo_2 + 1, foo_3);17;
10;statement;return fun_1(foo_1, foo_2, foo_3);18;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;8;
7;9;10;
8;4;
9;4;

File: cfgs/pythoncfgs/FIND_THE_FIRST_MISSING_NUMBER.py
Nodes
1;conditional;foo_1 > foo_2;8;
2;conditional;foo_1 != foo_2[foo_3];10;
3;statement;foo_1 = fun_1((foo_2 + foo_3) / 2);12;
4;conditional;foo_1[foo_2] == foo_3;13;
5;statement;return fun_1(foo_1, foo_2, foo_3);15;
6;statement;return fun_1(foo_1, foo_2 + 1, foo_3);14;
7;statement;return foo_1;11;
8;statement;return foo_1 + 1;9;
9;exit;;
Edges
0;2;8;
1;3;7;
2;4;
3;5;6;
4;9;
5;9;
6;9;
7;9;
8;
